resources/views/form-submission-uploads-failed.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+get_header(); ?>
+<main id="main" class="template-openzaak">
+    <div class="container">
+        <div class="container-inner">
+            <header>
+                <h1>
+                    Er ging iets fout!
+                </h1>
+                <div id="readspeaker_button1" class="rs_skip rsbtn rs_preserve">
+                    <a rel="nofollow" class="rsbtn_play" accesskey="L"
+                        title="Laat de tekst voorlezen met ReadSpeaker webReader"
+                        href="//app-eu.readspeaker.com/cgi-bin/rsent?customerid=8150&amp;lang=nl_nl&amp;readid=readspeaker&amp;url=<?php echo get_permalink() ?>">
```

Comment:
Deze templates kunnen worden overschreven vanuit het project toch? Ik denk dat we ReadSpeaker beter overal weg kunnen halen. Het id gaat alleen voor enkele sites werken en vormt ook problemen met SRI.